### PR TITLE
CON-#### Add `count-character` subject

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/count-character/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# count-character
```
Comment (Hamzaelkhatri) [inline subjects/count-character/README.md:1]: add `##`

File: subjects/count-character/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### Instructions
+write a function that takes a string and a character as arguments and returns the number of times the character appears in the string
+  
+    if the character is not in the string return 0
```
Comment (Hamzaelkhatri) [inline subjects/count-character/README.md:6]: add `-` in 
> if the cha...

File: subjects/count-character/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+### Usage
+
+```go
+    package main
+
+    import (   
+        "fmt"
+        "piscine"
```
Comment (Hamzaelkhatri) [inline subjects/count-character/README.md]: remove library `piscine`

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/count-character/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+}
+```
+
+And its output :
+
+```console
+$ go run .
+3
+0
+1
```
Comment (MSilva95) [inline subjects/count-character/README.md:42]: Isn't this supposed to return 3?

Comment (zainabdnaya) [reply subjects/count-character/README.md:42]: it's space and tabulation (\t)

Comment (zainabdnaya) [reply subjects/count-character/README.md:42]: And I m looking for space so there is only one

File: subjects/count-character/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+### Instructions
+write a function that takes a string and a character as arguments and returns the number of times the character appears in the string.
+- if the character is not in the string return 0
+- if the string is empty return 0
+
+### Expected Function
+
+```go
+func CountChar(str string, c rune) int {
+    ...
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/count-character/README.md]: done

**Review by MSilva95 — APPROVED** [review]

**Pull request closed without merging**